.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": []
```

Comment:
Consider having `.vscode` folder in `.gitignore` as its development environment specific and is not related to project itself. 🤔